winterfell/webapp/src/js/faqCtrl.js

```diff
@@ -1,20 +1,70 @@
 var app = angular.module('vetafiApp');
+
+app.filter('unsafe', function($sce) {
```

Comment:
this is used in `faq.pug`
```
div.section-content(ng-show="section.html", ng-bind-html="section.html | unsafe")
```
It's for injecting html content, but you can only do it with the $sce module (well one of the easiest way). Otherwise, Angular won't do it and will give back an error.
https://docs.angularjs.org/api/ng/service/$sce